Userland/Libraries/LibGfx/JPEGLoader.cpp

```diff
@@ -425,6 +426,17 @@ static inline ErrorOr<void> ensure_bounds_okay(const size_t cursor, const size_t
     return {};
 }
 
+static bool is_frame_marker(Marker const marker)
+{
+    // B.1.1.3 - Marker assignments
+    bool const is_sof_marker = marker >= JPEG_SOF0 && marker <= JPEG_SOF15;
+
+    // Start of frame markers are valid for JPEG_SOF0 to JPEG_SOF15 except number 4, 8 (reserved) and 12.
+    bool const is_defined_marker = marker != 0xFFC4 && marker != 0xFFC8 && marker != 0xFFCC;
```

Comment:
0XFFC4 is JPEG_DHT – should this use that name?

0xFFCC is JPEG_DAC which doesn't have a named constant in this file yet. Should it grow a name and should that be used here?

Reply:
I didn't even realize that these non-SOF markers were attributed.
I will use JPEG_DHT and add JPEG_DAC